cpp/include/cudf/ast/expressions.hpp

```diff
@@ -27,6 +27,24 @@
 #include <vector>
 
 namespace CUDF_EXPORT cudf {
+
+namespace detail {
+namespace row_ir {
+
+/// @brief The base class for all IR nodes.
+/// This class defines the interface for IR nodes, which can be instantiated and used to generate
+/// code. Each IR node represents a specific operation or value in the program. They represent a
+/// single-static-assignment (SSA) variable in the program IR. It is separate from the AST as it
+/// contains more detailed program information and analysis that would be needed to instantiate the
+/// program and generate correct and robust code.
```

Comment:
```suggestion
/**
 * @brief The base class for all IR nodes
 *
 * This class defines the interface for IR nodes, which can be instantiated and used to generate
 * code. Each IR node represents a specific operation or value in the program. They represent a
 * single-static-assignment (SSA) variable in the program IR. It is separate from the AST as it
 * contains more detailed program information and analysis that would be needed to instantiate the
 * program and generate correct and robust code.
 */
```
Use the format described in our documentation documentation: https://github.com/rapidsai/cudf/blob/branch-25.10/cpp/doxygen/developer_guide/DOCUMENTATION.md#the-example
The empty line after the `@brief` tag is important.